src/sidebar/components/annotation-viewer-content.js

```diff
@@ -8,9 +8,16 @@ import useRootThread from './hooks/use-root-thread';
 
 import ThreadList from './thread-list';
 import SidebarContentError from './sidebar-content-error';
+/**
```

Comment:
Can you ensure there is a new line after imports and before any JSDoc comments or other items.